### PR TITLE
Popup Author Information Functionality

### PR DESCRIPTION
**Pull request by coopermeitz**
It uses the first author's name to make a Wikipedia search. Currently returns the first sentence on the Wikipedia page, or for some reason prints "undefined" if a page does not exist.

image
image

### CODE REVIEW

**Comment by coopermeitz** [comment]
The package patches are no longer necessary because I stopped using that library

File: patches/wikibase-sdk+7.9.1.patch [hunk 1]
```diff
@@ -0,0 +1,73 @@
+diff --git a/node_modules/wikibase-sdk/lib/queries/get_entities_from_sitelinks.js b/node_modules/wikibase-sdk/lib/queries/get_entities_from_sitelinks.js
```
Comment (Phildesro123) [inline patches/wikibase-sdk+7.9.1.patch:1]: Do we need this file?

Comment (coopermeitz) [reply patches/wikibase-sdk+7.9.1.patch:1]: No, it's just my modification to the library before I stopped using it.

Comment (Phildesro123) [reply patches/wikibase-sdk+7.9.1.patch:1]: Its okay if I remove it?

Comment (coopermeitz) [reply patches/wikibase-sdk+7.9.1.patch:1]: Yep

**Review by Phildesro123 — APPROVED** [review]
Great work

**Pull request merged**